docs/docs/30-administration/00-deployment/00-overview.md

```diff
@@ -16,7 +16,11 @@ You can add more agents to increase the number of parallel workflows or set the
 
 Woodpecker is having two different kinds of releases: **stable** and **next**.
 
-To find out more about the differences between the two releases, please read the [FAQ](/faq#which-version-of-woodpecker-should-i-use).
+The **stable** releases (currently version 2.5) are stable versions. The stable releases are only getting bugfixes.
```

Comment:
Might not be the perfect place to explain our versioning just here as it is pretty deep in the docs.